src/backend/distributed/shared_library_init.c

```diff
@@ -584,6 +595,17 @@ RegisterCitusConfigVariables(void)
 		NULL, NULL, NULL);
 
 	DefineCustomEnumVariable(
+		"citus.replication_model",
+		gettext_noop("TODO."),
+		gettext_noop("TODO."),
```

Comment:
(I'm not 100% sure what the description of this GUC should be. I gave it a best-effort, but let me know if I should change anything).